prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -104,11 +104,12 @@ object FhirPathUtils : Logging {
             }
         } catch (e: Exception) {
             // This is due to a bug in at least the extension() function
-            val msg =  when (e) {
+            val msg = when (e) {
                 is FHIRLexerException -> "Syntax error in FHIR Path expression $expression"
                 is SchemaException -> e.message ?: "Condition error in FHIR Path expression $expression"
-                else -> "Unknown error while evaluating FHIR Path expression $expression for condition. " +
-                    "Setting value of condition to false."
+                else ->
```

Comment:
Try merging `master` in; I believe this is what I saw yesterday and put the PR up for